geps/gep-1897/index.md

```diff
@@ -223,7 +225,11 @@ between the gateway and backend pod. References to a resource in a different nam
 If ClientCertificateRefs is unspecified, then WellKnownCACertificates must be set to "System" for a valid configuration.
 If WellKnownCACertificates is unspecified, then CACertificateRefs must be specified with at least one entry for a valid configuration.
 If WellKnownCACertificates is set to "System" and there are no system trusted certificates or the implementation doesn't define system
-trusted certificates, then the associated TLS connection must fail.
+trusted certificates, the BackendTLSPolicy is considered invalid.
```

Comment:
Again, I don't think we can insist on this level of validation except at runtime.  There may not be a system trusted certificate when the policy is created, but we shouldn't mark it invalid unless the connection fails because of that.  Alternatively, there may be a system trusted certificate when it is created, but it expires at some time in the future and makes the connection fail.  

I don't think we want the implementations to set up watches on these certificates.

Reply:
Fair question, but how is the implementation supposed to determine whether WellKnownCACertificates are supported at the time when it sets the status for the BackendTLSPolicy?

Reply:
I think we might be talking past each other a bit. What I was referring to is already captured in the API specification:
```
// If an implementation does not support the WellKnownCACertificates field,
// or the provided value is unsupported, the Policy's Status Conditions MUST be
// updated to include an Accepted: False Condition with Reason: Invalid.
```
https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1alpha3/backendtlspolicy_types.go#L140